src/qt/coincontroldialog.cpp

```diff
@@ -654,15 +821,53 @@ void CoinControlDialog::updateView()
             }
 
             // amount
+            uint64_t nBlockSize = out.tx->vout[out.i].nValue / COIN;
             itemOutput->setText(COLUMN_AMOUNT, BitcoinUnits::format(nDisplayUnit, out.tx->vout[out.i].nValue));
             itemOutput->setText(COLUMN_AMOUNT_INT64, strPad(QString::number(out.tx->vout[out.i].nValue), 15, " ")); // padding so that sorting works correctly
 
             // date
-            itemOutput->setText(COLUMN_DATE, QDateTime::fromTime_t(out.tx->GetTxTime()).toUTC().toString("yy-MM-dd hh:mm"));
-            
+            uint64_t nTime = out.tx->GetTxTime();
+            itemOutput->setText(COLUMN_DATE, GUIUtil::dateTimeStr(QDateTime::fromTime_t(nTime)));
+              		              
+            // immature PoS reward
+            if (out.tx->IsCoinStake() && out.tx->GetBlocksToMaturity() > 0 && out.tx->GetDepthInMainChain() > 0) {
+                itemOutput->setBackground(COLUMN_CONFIRMATIONS, Qt::red);
+                itemOutput->setDisabled(true);
+            }
+
             // confirmations
             itemOutput->setText(COLUMN_CONFIRMATIONS, strPad(QString::number(out.nDepth), 8, " "));
             
+            nInputSum += nInputSize;
+
+            // List mode weight
+            itemOutput->setText(COLUMN_WEIGHT, strPad(QString::number(nDisplayWeight), 8, " "));
+
+            // age
+            uint64_t nAge = (GetTime() - nTime);
+            itemOutput->setText(COLUMN_AGE, QString::number((double)nAge / 86400, 'f', 2));
+            itemOutput->setText(COLUMN_AGE_INT64, QString::number((double)nAge / 86400, 'f', 2));
+
+            // potential stake
+            double nPotentialStake = (double)nBlockSize * (GetCoinYearReward(nBestHeight)/CENT/100) * ((double)nAge / (86400) / 365);
```

Comment:
@konez2k I pulled this over and it is the last thing that is not compiling in this folder. You think we should get the APR/Maturation fix merged before we get this feature released so that we can create a better function to GetCoinYearReward?